### ISSUE TITLE
Add web archive for google.co.in

### ISSUE THREAD

**Issue by asajeffrey** [servo/servo-warc-tests#16]
#8

**Comment by sjoshid** [comment]
Hi @asajeffrey,
I would like to try this.

**Comment by sjoshid** [comment]
Running 
`proxychains ./mach run -r --certificate-path proxy-certs/pywb-ca.pem https://www.google.co.in`
gives me DNS error. 
```
ProxyChains-3.1 (http://proxychains.sf.net)
|DNS-request| www.google.co.in 
|S-chain|-<>-127.0.0.1:9050-<--timeout
|DNS-response|: www.google.co.in does not exist
```
Let me know if Im missing something. 
Also I get following error on Servo
![image](https://user-images.githubusercontent.com/6751090/36346730-c7b28956-1411-11e8-8f7e-5b077147127e.png)

**Comment by jdm** [comment]
https://github.com/servo/servo-warc-tests/pull/22/ contains updates to the readme that may help.

**Comment by asajeffrey** [comment]
@sjoshid Hi, thanks for taking this on! The error you're seeing is probably from running the command outside the servo-warcs-test directory, so it's not picking up the configuration file.

**Comment by sjoshid** [comment]
@jdm if you talking about adding --port 8321 when starting wayback then yes. I did add it. 
@asajeffrey  where exactly is this servo-warcs-test directory? 

Thanks both for replying. But it seems my proxychains is not working as expected. A simple command like this
`proxychains firefox www.google.com` 
gives me same DNS errors. 
I toyed around with proxyresolv, changing DNS_SERVER to Google's and others. I also reinstalled proxychains. But no luck. 
Any other suggestions would be appreciated. 

Thanks.

**Comment by asajeffrey** [comment]
The servo-warc-tests directory contains a config file that sets the options needed by proxychains. It's picked up automatically if you run proxychains from inside the directory that you cloned the repo into.

**Comment by sjoshid** [comment]
Yes. I have been running the proxychains command from the inside the repo directory all this time.

**Comment by asajeffrey** [comment]
Hmm, that's strange. Your copy of the repo has the file https://github.com/servo/servo-warc-tests/blob/master/proxychains.conf?

**Comment by sjoshid** [comment]
I cannot access my machine now but I dont remember seeing it. I'll take a fresh checkout and try again.

**Comment by asajeffrey** [comment]
Ah, that might be the issue!

**Comment by asajeffrey** [comment]
@sjoshid did you get this sorted out?

**Comment by sjoshid** [comment]
Yes. I was able to. Thank you.

**Comment by asajeffrey** [comment]
Closed by #29.
